### PR TITLE
[TASK] Improve dashboard view without  widgets

### LINKED ISSUE #64: Improve start screen if no widgets available
If no widgets used, the screen is pretty empty and people might not know what to do, please add a nice callout with proper CTA to add widgets

### PR DESCRIPTION
**Pull request by NeoBlack**
fixes #64

### CODE REVIEW

File: Resources/Private/Language/locallang.xlf [hunk 1]
```diff
@@ -97,6 +97,16 @@
 			<trans-unit id="widgets.documentation.label" xml:space="preserve">
 				<source>Check TYPO3 documentation</source>
  			</trans-unit>
+
+			<trans-unit id="dashboard.empty.title" xml:space="preserve">
+				<source>No dashboard configured :(</source>
```
Comment (haassie) [inline Resources/Private/Language/locallang.xlf]: I guess this should be: No widgets on this dashboard :(

Comment (NeoBlack) [reply Resources/Private/Language/locallang.xlf]: updated

**Comment by NeoBlack** [comment]
please merge first the widget registration refactoring, then I can/will rebase this one if required

**Comment by haassie** [comment]
This branch can be rebased

**Pull request merged**